### PR TITLE
creator autosync part 2

### PR DESCRIPTION
**Pull request by Robinnnnn**
### Summary of Changes

Provide a brief description of the changes made in this PR. Explain the problem you are fixing or the feature you are adding.

### Demo or Before/After Pics

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots.

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
|  |  |

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/AnimatedRefreshIcon.tsx [hunk 1]
```diff
@@ -26,13 +26,35 @@ export function AnimatedRefreshIcon({
...
-    onRefresh();
     pushToast({
-      message: 'Successfully refreshed your collection',
+      message: 'Refreshing your collection. This may take a minute!',
       withoutNavbar: true,
     });
-  }, [isSyncing, onSync, onRefresh, pushToast]);
+
+    /**
+     * NOTE: there's a SUPER FUCKING WEIRD UI bug here!
```
Comment (kaitoo1) [inline apps/mobile/src/components/AnimatedRefreshIcon.tsx:35]: damn what the hell
nice catch

**Review by kaitoo1 — APPROVED** [review]
bravo sir
